lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -1,6 +1,12 @@
 <%! from django.utils.translation import ugettext as _ %>
 <%page args="section_data"/>
 
+<script>
+  ${d3_graph_data_download.body()}
+</script>
+
+<%namespace name="d3_graph_data_download" file="/class_dashboard/d3_graph_data_download.js"/>
```

Comment:
class_dashboard is the metrics tab. Just wondering if there's a better place to put the file.